xml/depl_maintenance.xml

```diff
@@ -312,6 +312,175 @@
+     <systemitem>pacemaker_remote</systemitem> service on the &compnode;. Use
+     standard network debugging methods to make sure that nothing is preventing
+     this particular TCP connection from being established (for example, problems with
+     NICs, switches, firewalls etc.).
+    </para>
+   </step>
+   <step>
+    <para>
+     If Pacemaker can communicate with the remote node, it starts the
+     <systemitem>nova-compute</systemitem> service on it as part of the cloned
+     group <literal>cl-g-nova-compute</literal> using the NovaCompute OCF
+     resource agent. This cloned group will block startup of
+     <systemitem>nova-evacuate</systemitem> until at least one clone is
+     started. Therefore the next section is necessary as a related but
+     different procedure. See also the next step.
+    </para>
+   </step>
+   <step>
+    <para>
+     It may happen that <systemitem>NovaCompute</systemitem> has been launched
+     correctly on the &compnode; by <systemitem>lrmd</systemitem>, but the
+     <systemitem>openstack-nova-compute</systemitem> service is still not
+     running. This usually happens when <systemitem>nova-evacuate</systemitem>
+     did not run correctly. If <systemitem>nova-evacuate</systemitem> is not
+     running on one of the core cluster nodes, make sure that the service is
+     marked as started (<literal>target-role="Started"</literal>). If this is
+     the case, then your cloud probably does not meet the assumption of this
+     procedure. In case <systemitem>nova-evacuate</systemitem> is started but
+     it is failing, check the Pacemaker logs to determine the cause. When
+     <systemitem>nova-evacuate</systemitem> is started and functioning
+     correctly, it should call nova's evacuate API to release resources
+     used by the &compnode; and resurrect elsewhere any VMs that died when it
+     failed.   
+    </para>
+   </step>
+   <step>
+    <para>
+     If <systemitem>openstack-nova-compute</systemitem> is running, but VMs are
+     not booted on the node, check whether the service is not disabled or
+     forced down using the <command>nova service-list</command> command. In case the
+     service is disabled, run the <command>nova service-enable
+     $service_id</command> command. If the service is forced down, run the
+     following command:
+    </para>
+<screen>fence_nova_param () {
+    key="$1"
+    cibadmin -Q -A "//primitive[@id='fence-nova']//nvpair[@name='$key']" | \
+    sed -n '/.*value="/{s///;s/".*//;p}'
+}
+fence_compute \
+    --auth-url=`fence_nova_param auth-url` \
+    --endpoint-type=`fence_nova_param endpoint-type` \
+    --tenant-name=`fence_nova_param tenant-name` \
+    --domain=`fence_nova_param domain` \
+    --username=`fence_nova_param login` \
+    --password=`fence_nova_param passwd` \
+    -n $compute_hostname \
+    --action=on
+</screen>
+   </step>
+  </procedure>
+  <para>
+   The described steps should be performed automatically after the node is
```

Comment:
"described steps" sounds both unclear and a bit odd.

Reply:
Would you prefer _these steps_, _the steps above_, something else?